pkgs/development/libraries/binutils-opcodes/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv, buildPackages
+, fetchurl, autoreconfHook264, bison, binutils
+, libiberty, libbfd
+}:
+
+stdenv.mkDerivation rec {
+  name = "binutils-opcodes-${version}";
+  inherit (binutils) version src;
+
+  outputs = [ "out" "dev" ];
+
+  patches = binutils.patches ++ [
+    ../../tools/misc/binutils/build-components-separately.patch
+  ];
+
+  # We just want to build binutils-opcodes
+  postPatch = ''
+    cd opcodes
+    find . ../include/opcode -type f -exec sed {} -i -e 's/"bfd.h"/<bfd.h>/' \;
+  '';
+
+  nativeBuildInputs = [ autoreconfHook264 bison buildPackages.stdenv.cc ];
```

Comment:
Is `buildPackages.stdenv.cc` necessary here? Isn't this the default?

Reply:
Nah that would be `stdenv.cc` (or `pkgs.stdenv.cc`). This is for building the codegen tools.